docs/guides/latest-updates.mdx

```diff
@@ -1,271 +1,278 @@
+- For full details on the early access release and upcoming improvements, [check out the blog.](https://www.ibm.com/quantum/blog/iqp-upgrade)
+</CloudContent>
 
-- We improved performance by porting additional key components to Rust, including:
-  - Core data structures, such as `DAGCircuit` or `Target`
-  - Transpiler passes, including `BasisTranslator`, `CommutationAnalysis`, or `UnitarySynthesis`, making the transpilation around six times faster compared to v1.2.4
-  - Circuit library and synthesis routines, such as `PauliEvolutionGate` or `n_local`
+<span id="latest-qiskit-sdk"></span>
+## Qiskit SDK v1.4.0
 
-- The circuit library is now faster and allows for better high-level optimization, in particular with context-aware updates to `HighLevelSynthesis`.
+**21 Feb 2025**
 
-- Python 3.13 is now officially supported. Python 3.9 is now the *minimum* supported version.
+- The latest Qiskit version, 1.4.0, has been released on GitHub and is available on [PyPi](https://pypi.org/project/qiskit/). Even if you recently upgraded to v1.3.3, you should still upgrade to Qiskit v1.4, because this is the last minor version of the v1.x series, and contains all of the deprecation warnings for API changes in Qiskit v2.0. Install it now, and see what to change to be ready for 2.0!
 
-Explore the [v1.3 release notes](/api/qiskit/release-notes/1.3), and read about this release on the [IBM Quantum blog](https://www.ibm.com/quantum/blog/qiskit-1-3-release-summary).
+  Wondering what will change? Read the [changelog](https://github.com/Qiskit/qiskit/releases/tag/1.4.0) and the [v1.4 release notes](/api/qiskit/release-notes/1.4).
 
 {/* use url format /api/qiskit/release-notes/X.X to link to the specific X.X version of the release notes */}
 
-### Qiskit Runtime client 0.37.0
+- *To see the release notes for all versions, visit the [Qiskit SDK release notes](/api/qiskit/release-notes).*
+
 
-*To see the release notes for all versions, visit the [Qiskit Runtime client release notes](/api/qiskit-ibm-runtime/release-notes).*
+<span id="latest-qiskit-runtime"></span>
+## Qiskit Runtime client 0.37.0
 
-**Highlights from the 0.37.0 (13 March 2025) version release**
+**13 March 2025**
 
-#### Deprecation
+- Deprecation
 
-The `IBMBackend` attributes `max_shots` and `max_experiments`, as well as the property `max_circuits` have all been deprecated. Previously, these attribute values represented the maximum number of shots and circuits that could be submitted in one job. See [job limits](/guides/job-limits#job-limits) for details.
+  - The `IBMBackend` attributes `max_shots` and `max_experiments`, as well as the property `max_circuits`, have all been deprecated. Previously, these attribute values represented the maximum number of shots and circuits that could be submitted in one job. See [job limits](/guides/job-limits#job-limits) for details.
```

Comment:
Can we get rid of these single bullets and just have them be a regular paragraph?